### PR TITLE
fix: Change CI to Jenkins and bump Axios

### LINKED ISSUE #110: Replace CirlcleCI with Jenkins
We need to keep this repository up-to-date enough with CI to ensure that we can apply patches.  Synk has identified that Axios needs to be upgraded.

* Change to Jenkins

### LINKED ISSUE #111: Bump Axios to > `0.25.0`
Synkbot has recommended upgrading Axios due to some bug/security patches.  We will do this manually in a new issue since the recommendation was blocked by #110

### PR DESCRIPTION
**Pull request by darinspivey**
**chore(deps): axios@0.25.0**

Fixes: #111

---

**fix(ci): Switch to Jenkins and semantic-release**

Use Jenkins for CI and semantic-release for packaging.

Fixes: #110

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,141 @@
+library 'magic-butler-catalogue'
+
+def PROJECT_NAME = "nodejs_client"
```
Comment (mdeltito) [inline Jenkinsfile]: maybe `nodejs-client-legacy` or something?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        beforeAgent true
+        branch DEFAULT_BRANCH
+        not {
+          changelog '\\[skip ci\\]'
+        }
+      }
+
+      agent {
+        docker {
+          image "us.gcr.io/logdna-k8s/node:12-ci"
```
Comment (mdeltito) [inline Jenkinsfile]: might as well make this `14-ci` too

**Review by mdeltito — DISMISSED** [review]
👍  comments are non-blocking

**Review by aholmberg — DISMISSED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (darinspivey) [reply Jenkinsfile]: yeah, fair.

**Review by aholmberg — APPROVED** [review]

**Review by mdeltito — APPROVED** [review]

**Pull request merged**